eve-tools/bpftrace-compiler/getty/bpftrace-helper/bpftrace-helper.go

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
+
+func allWriter() *os.File {
+	writers := make([]*os.File, 0)
+
+	devs := []struct {
+		path  string
+		major uint32
+		minor uint32
+	}{
+		{
+			path:  "/dev/ttyS0",
```

Comment:
are you trying to make "every possible" console available? We have a fair list here: https://github.com/lf-edge/eve/blob/master/images/rootfs.yml.in#L109